desktop/core/src/desktop/js/utils/url/changeURLParameter.ts

```diff
@@ -27,8 +27,13 @@ const changeURLParameter = (param: string, value: string | null): void => {
       .forEach(p => {
         if (p.split('=')[0] !== param) {
           newSearch += p;
+          newSearch += '&';
         }
       });
+    if (newSearch !== '?') {
+      newSearch = newSearch.substr(0, newSearch.length -1);
+    }
```

Comment:
Would it make sense to add the '&' in the foreach if newSearch != '?'
?
(easier to understand this way)

Reply:
nice!